packages/busybox/busybox.config

```diff
@@ -950,15 +950,15 @@ CONFIG_NC_110_COMPAT=y
 CONFIG_NETSTAT=y
 CONFIG_FEATURE_NETSTAT_WIDE=y
 CONFIG_FEATURE_NETSTAT_PRG=y
-# CONFIG_NSLOOKUP is not set
-# CONFIG_FEATURE_NSLOOKUP_BIG is not set
-# CONFIG_FEATURE_NSLOOKUP_LONG_OPTIONS is not set
-# CONFIG_NTPD is not set
+CONFIG_NSLOOKUP=n
+CONFIG_FEATURE_NSLOOKUP_BIG=n
+CONFIG_FEATURE_NSLOOKUP_LONG_OPTIONS=n
```

Comment:
Is there a particular reason to set these to `n` instead of leaving them commented out?

Reply:
Sorry, I made a mistake.